client/jetpack-cloud/sections/partner-portal/hooks.ts

```diff
@@ -38,3 +38,62 @@ export function useRecentPaymentMethodsQuery( { enabled = true }: UseQueryOption
 		}
 	);
 }
+
+/**
+ * Returns false; if flag ever becomes true, then returns true from then onwards.
```

Comment:
This sounds a bit like the `@returns` description. Should it perhaps be something more generic like: `Handle permanent state of has more` ?

Reply:
I didn't want to specifically tie it to hasMore because it's a generic function even if its current functionality is meant for hasMore use.